docs/src/inputs/input/devSandbox/devSandbox.jsx

```diff
@@ -0,0 +1,565 @@
+                    >
+                        onFocus Event Handler
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Can handle an <code>onFocus</code> event from parent.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputOnFocus').default}
+                >
+                    <ExampleInputOnFocus />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        onKeyDown Event Handler
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Can handle an <code>onKeyDown</code> event from parent.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputOnKeyDown').default}
+                >
+                    <ExampleInputOnKeyDown />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Placeholder
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A hint to the user of what can be entered in the input.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputPlaceholder').default}
+                >
+                    <ExampleInputPlaceholder />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Required
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Specifies that the user must fill in a value before submitting a form.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputRequired').default}
+                >
+                    <ExampleInputRequired />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Type
+                    </Heading>
```

Comment:
`anchorLink` should be something like `"type"`

Reply:
Fixed. Please check this [Commit](https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/46c752aa0e781aedba48ebab9e7a168431118aa2)

Reply:
Looks good.  Resolving.